src/data/equiv/algebra.lean

```diff
@@ -328,3 +327,73 @@ instance symm.is_ring_hom {e : α ≃r β} : is_ring_hom e.to_equiv.symm := hom
   e.symm.to_equiv x = e.to_equiv.symm x := rfl
 
 end ring_equiv
+
+-- We first define the type of automorphisms on groups, additive monoids,
+-- additive groups and rings using `mul_equiv`, `add_equiv` and `ring_equiv`.
+-- In each case this type is also a group.
+
+namespace group
+
+def aut (γ : Type) [group γ] := mul_equiv γ γ
+
+@[extensionality] lemma ext_mul_equiv (α β : Type) [group α] [group β] (f : mul_equiv α β)
```

Comment:
Make arguments which can be implicit, implicit. In this case, `α β f g` should all be in `{ }`.

Reply:
fixed